### ISSUE TITLE
DOCKER_LOG fails if transmission isn't running as root

### ISSUE THREAD

**Issue by t0xicCode** [haugene/docker-transmission-openvpn#1117]
DOCKER_LOG fails when transmission is not running as root because it tries to open `/proc/1/fd/1`, which is chown'd to `root`.

Essentially, dumb-init runs as root, which makes `/proc/1/fd/1` owned by root. If transmission is ran as any other user, it is unable to open that file, as it lacks the permissions.

```sh
docker run --cap-add=NET_ADMIN -e DOCKER_LOG=true -e PGID=1234 -e PUID=1234 --name "log-fail-demo" haugene/transmission-openvpn:latest
```

Relevant log section:
```
Transmission startup script complete.
Fri Apr  3 15:27:23 2020 Initialization Sequence Completed
Couldn't (re)open log file "/proc/1/fd/1": Permission denied
```

Running on an amd64 NAS.

**Comment by pkishino** [comment]
Hi, thanks for the feedback, I’ll try and look into this when I have time. Should be possible to change so it’s usable by non-root

**Comment by wilmardo** [comment]
@pkishino Sorry I didn't mean to snatch your contribution but I ran into this myself yesterday and fixed it ;)

**Comment by pkishino** [comment]
@wilmardo  hey, thanks, looks good :) great work. I tried using stdout originally but had some issues, great if this works :)

**Comment by wilmardo** [comment]
@pkishino If you would like to test, I build and pushed the alpine for testing:
https://hub.docker.com/r/lansibledump/transmission/tags

So just use `lansibledump/transmission:latest` instead of `haugene/transmission-openvpn`

**Comment by pkishino** [comment]
This is working now I believe, please check and close

**Comment by pkishino** [comment]
@t0xicCode Please close this as this was fixed in #1122

**Comment by t0xicCode** [comment]
I can confirm that it was fixed. Thanks y’all
